code/modules/surgery/organs/autoimplanter.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/device/autoimplanter
+	name = "autoimplanter"
+	desc = "A device that automatically injects a cyber-implant into the user without the hassle of extensive surgery. It has a slot to insert implants and a screwdriver slot for removing accidentally added implants."
+	icon_state = "gangtool-white"
+	item_state = "walkietalkie"
+	w_class = 2
+	var/obj/item/organ/internal/cyberimp/storedorgan
+
+/obj/item/device/autoimplanter/attack_self(mob/user)//when the object it used...
+	if(!storedorgan)
+		user << "<span class='notice'>[src] currently has no implant stored.</span>"
+		return
+	storedorgan.Insert(user)//insert stored organ into the user
+	user.visible_message("<span class='notice'>[user] presses a button on [src], and you hear a short mechanical noise.</span>")
```

Comment:
Oops, I meant to specify a message for both... For instance:
`user.visible_message("<span class='notice'>[user] presses a button on [src], and you hear a short mechanical noise.</span>", "<span class='notice'>You feel a sharp sting as [src] plunges into your body.</span>")`